backend/api/alerts.py

```
@@ -74,33 +63,8 @@ def get_alert_summary(
     to_date: str,
     batches: Optional[str] = "",
     severities: Optional[str] = "",
-    credentials: HTTPAuthorizationCredentials = Depends(security)
+    credentials: HTTPAuthorizationCredentials = Depends(security),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)